keyboards/xd75/keymaps/hybrid/keymap.c

```diff
@@ -0,0 +1,310 @@
+/* Copyright 2020 Hybrid
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+
+enum layers {
+    _MAIN = 0,
+    _LY1,
+    _LY2,
+};
+
+enum {
+  TD_A,
+  TD_E,
+  TD_I,
+  TD_O,
+  TD_U,
+  TD_SLASH,
+  TD_UNDER,
+  TD_OP_ACCOL,
+  TD_CL_ACCOL,
+  TD_QUESTION,
+  TD_CEDILLE,
+  TD_DOT,
+};
+
+
+
+void dance_a_accent (qk_tap_dance_state_t *state, void *user_data) {
+  if (state->count == 2) {
+    tap_code (KC_0);
+  }
+  else {
+    tap_code (KC_Q);
+  }
+}
+void dance_e_accent (qk_tap_dance_state_t *state, void *user_data) {
+  if (state->count == 2) {
+    tap_code (KC_2);
+  }
+  else if (state->count == 3) {
+    tap_code (KC_7);
+  }
+  else {
+    tap_code (KC_E);
+  }
+}
+void dance_i_accent (qk_tap_dance_state_t *state, void *user_data) {
+  if (state->count == 2) {
+    tap_code (KC_NLCK);
+    register_code (KC_LALT);
+    tap_code (KC_P0);
+    tap_code (KC_P2);
+    tap_code (KC_P3);
+    tap_code (KC_P8);
+    unregister_code (KC_LALT);
+    tap_code (KC_NLCK);
```

Comment:
Just a heads up, you could use Unicode and the [`send_unicode_string`](https://docs.qmk.fm/#/feature_unicode?id=send_unicode_string) or [`send_unicode_hex_string`](https://docs.qmk.fm/#/feature_unicode?id=send_unicode_hex_string).